### PR TITLE
Add(FileInfo): acquisionVersion, ampType

### PR DESCRIPTION
**Pull request by jusjusjus**

### CODE REVIEW

File: mffpy/tests/test_xml_files.py [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
 @pytest.fixture
 def file_info():
-    ans = join(mff_path, 'info.xml')
+    ans = join(examples_path, 'example_2.mff', 'info.xml')
```
Comment (jusjusjus) [inline mffpy/tests/test_xml_files.py:41]: example_2.mff has acquisitionVersion and ampType, example_1.mff doesn't.

**Review by ephathaway — CHANGES_REQUESTED** [review]
Just a few documentation things, but otherwise looks good!

File: mffpy/xml_files.py [hunk 2]
```diff
@@ -153,42 +154,73 @@ class FileInfo(XML):
...
 
+    @property
+    def version(self):
+        """return mffVersion"""
+        warnings.warn(".version is deprecated, use .mffVersion instead",
+                      DeprecationWarning)
+        return self.mffVersion
+
+    @cached_property
+    def acquisitionVersion(self):
```
[the remark this answers is not in the document]
Comment (jusjusjus) [reply mffpy/xml_files.py:169]: Well, if you have an idea of what that is; I don't.

**Comment by jusjusjus** [comment]
Thanks for the review; I submitted more changes to the doc strings.

File: mffpy/xml_files.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ephathaway) [reply mffpy/xml_files.py:169]: I think this is the version of Net Station used to record the data.

**Review by ephathaway — APPROVED** [review]
Looks good! Thanks for adding those docstrings.

**Pull request merged**